widget/embedded/src/utils/common.ts

```diff
@@ -81,3 +83,40 @@ export const generateRangeColors = (name: string, color: string) => {
 
 export const getContainer = () =>
   document.getElementById('swap-box') as HTMLElement;
+
+export const generateColors = (
+  mainColors: { [x: string]: string },
+  colors?: WidgetColors
+) => {
```

Comment:
Please move this method to another util file for colors.

Reply:
fixed